src/methods.cpp

```diff
@@ -1528,7 +1541,7 @@ PyObject* meth_flash_devices(PyObject* self, PyObject* args)
                                     void (*MessageCallback)(const char* message, bool success))>
             FlashDevice2(lib, "FlashDevice2");
         Py_BEGIN_ALLOW_THREADS;
-        if (!FlashDevice2(0x3835C256, &nde->neoDevice, rc, reflash_count, 0, 0, 0, &message_callback)) {
+        if (!FlashDevice2(0x3835C256, &nde->neoDevice, rc, reflash_count, network_id, 0, 0, &message_callback)) {
```

Comment:
looks like we are missing iOptions

Reply:
adding the options will be there my next push

Reply:
This needs to be updated still.